common/src/main/java/com/mx/path/core/common/connect/ResponseProcessingException.java

```diff
@@ -0,0 +1,12 @@
+package com.mx.path.core.common.connect;
+
+import com.mx.path.core.common.accessor.UpstreamSystemUnavailable;
+
+/**
+ * Wraps unexpected exception type thrown during Response processing
+ */
+public final class ResponseProcessingException extends UpstreamSystemUnavailable {
```

Comment:
What's the reason behind making this extend `UpstreamSystemUnavailable` instead of `AccessorSystemException`?

Reply:
I was trying to keep existing behavior, but I think you're right. In the case where a random (non-Path) exception is raised from the Response, or Response.process block, there is a pretty big problem with our code. Fixed.